.env

```diff
@@ -0,0 +1,2 @@
+GROQ_API_KEY=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
```

Comment:
Your `.env` file should be added to gitignore. It has secrets that shouldn't be public.